### PR TITLE
Add a helper script used to compare two L0 FITS files

### PR DESCRIPTION
**Pull request by jeanconn**
Add a helper script used to compare two L0 FITS files

### CODE REVIEW

**Comment by jeanconn** [comment]
@taldcroft  if you see anything important that that I might have forgotten to check when comparing these two files, please let me know.

**Comment by jeanconn** [comment]
```
In [87]: run check_file.py
Checking first hdu header lines
H1: SIMPLE  =                    T /Primary Header created by MWRFITS v1.2          
H2: SIMPLE  =                    T /Primary Header created by MWRFITS v1.13         
Checking second hdu header lines
H1: XTENSION= 'BINTABLE'           /Binary table written by MWRFITS v1.2            
H2: XTENSION= 'BINTABLE'           /Binary table written by MWRFITS v1.13           
H1: TTYPE1  = 'TIME_ERT '          /                                                
H2: TTYPE1  = 'TIME_ERT'           /                                                
H1: TTYPE3  = 'ADC_BIAS '          /                                                
H2: TTYPE3  = 'ADC_BIAS'           /                                                
H1: TTYPE9  = 'SCI_HDR_CTR '       /                                                
H2: TTYPE9  = 'SCI_HDR_CTR'        /                                                
H1: TTYPE10 = 'TWO_US_CLK '        /                                                
H2: TTYPE10 = 'TWO_US_CLK'         /                                                
Checking file data in all cols but PIXDATA
Checking data in PIXDATA col
Done
```

**Comment by taldcroft** [comment]
Would astropy `fitsdiff` (http://docs.astropy.org/en/stable/io/fits/api/diff.html) work?

**Comment by jeanconn** [comment]
That looks like it would simplify things,  thanks!  I was focused on not
using CIAO dmdiff and forgot to look for better tools in astropy.

**Comment by jeanconn** [comment]
Though I don't know if my simple loop over the PIXDATA might end up being
more of the kind of simple check we could want on intermediate data.  But
that probably getting ahead.

File: check_file.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    if col == 'PIXDATA':
+        continue
+    for o_val, n_val in zip(o_data[col], n_data[col]):
+        if o_val != n_val:
+            print "orig: {}".format(o_val)
+            print "new : {}".format(n_val)
+
+print "Checking data in PIXDATA col"
+for o_row, n_row in zip(o_data['PIXDATA'], n_data['PIXDATA']):
+    if not np.all(o_row == n_row):
```
Comment (taldcroft) [inline check_file.py:46]: Might want some variant of `allclose` since there are likely to be small floating point diffs beteween IDL and Python.

Comment (jeanconn) [inline check_file.py:46]: The PIXDATA values are ints.  I think we might get away with exact matches in those.  It looks like the times are floats, so we'll probably need to `allclose` those.

**Pull request merged**